assets/xml/objects/object_skb.xml

```diff
@@ -1,64 +1,65 @@
-        <Skeleton Name="object_skb_Skel_005EF8" Type="Normal" LimbType="Standard" LimbNone="OBJECT_SKB_LIMB_NONE" LimbMax="OBJECT_SKB_LIMB_MAX" EnumName="ObjectSkbLimb" Offset="0x5EF8" />
-        <Animation Name="object_skb_Anim_0064E0" Offset="0x64E0" />
+        <Animation Name="gStalchildStandUpAnim" Offset="0x3584" />
+        <Texture Name="gStalchildPelvisTex" OutName="stalchild_pelvis" Format="rgba16" Width="16" Height="16" Offset="0x35A0" />
+        <Texture Name="gStalchildRibcageTex" OutName="stalchild_ribcage" Format="rgba16" Width="8" Height="16" Offset="0x37A0" />
+        <Texture Name="gStalchildSkullTex" OutName="stalchild_skull" Format="rgba16" Width="8" Height="8" Offset="0x38A0" />
+        <Texture Name="gStalchildNoseTex" OutName="stalchild_nose" Format="rgba16" Width="8" Height="8" Offset="0x3920" />
+        <Texture Name="gStalchildSingleBoneTex" OutName="stalchild_single_bone" Format="rgba16" Width="8" Height="8" Offset="0x39A0" />
+        <Texture Name="gStalchildHandFootTex" OutName="stalchild_hand_foot" Format="rgba16" Width="8" Height="16" Offset="0x3A20" />
+        <Texture Name="gStalchildJawTex" OutName="stalchild_jaw" Format="rgba16" Width="16" Height="8" Offset="0x3B20" />
+        <Texture Name="gStalchildEyeSocketTex" OutName="stalchild_eye_socket" Format="rgba16" Width="16" Height="16" Offset="0x3C20" />
+        <Texture Name="gStalchildEyeTex" OutName="stalchild_eye" Format="rgba16" Width="8" Height="8" Offset="0x3E20" />
+        <DList Name="gStalchildHeadDL" Offset="0x4F30" />
+        <DList Name="gStalchildLowerJawDL" Offset="0x5298" />
+        <DList Name="gStalchildRibcageDL" Offset="0x5340" />
+        <DList Name="gStalchildSpineDL" Offset="0x5408" />
+        <DList Name="gStalchildPelvisDL" Offset="0x54A0" />
+        <DList Name="gStalchildLeftUpperArmDL" Offset="0x55A8" />
+        <DList Name="gStalchildLeftLowerArmDL" Offset="0x5668" />
+        <DList Name="gStalchildLeftHandDL" Offset="0x5710" />
+        <DList Name="gStalchildRightUpperArmDL" Offset="0x57D8" />
+        <DList Name="gStalchildRightLowerArmDL" Offset="0x5898" />
+        <DList Name="gStalchildRightHandDL" Offset="0x5940" />
+        <DList Name="gStalchildLeftFemurDL" Offset="0x5A08" />
+        <DList Name="gStalchildLeftShinDL" Offset="0x5AA0" />
+        <DList Name="gStalchildLeftFootDL" Offset="0x5B48" />
+        <DList Name="gStalchildRightFemurDL" Offset="0x5BE8" />
+        <DList Name="gStalchildRightShinDL" Offset="0x5C80" />
+        <DList Name="gStalchildRightFootDL" Offset="0x5D28" />
+        <Limb Name="gStalchildRootLimb" Type="Standard" EnumName="STALCHILD_LIMB_ROOT" Offset="0x5DC8" />
+        <Limb Name="gStalchildPelvisLimb" Type="Standard" EnumName="STALCHILD_LIMB_PELVIS" Offset="0x5DD4" />
+        <Limb Name="gStalchildLegsRootLimb" Type="Standard" EnumName="STALCHILD_LIMB_LEGS_ROOT" Offset="0x5DE0" />
```

Comment:
OoT has this as Root2Limb, not saying this is inherently wrong, just posing a question for discussion for naming.

Reply:
I think it's probably best to match OoT, so I made this change. In retrospect I should have cheated and looked at their version of this file first >.<